mopidy_gmusic/library.py

```diff
@@ -88,6 +91,7 @@ def _browse_artist(self, uri):
             refs.sort(key=lambda ref: ref.name)
         if len(refs) > 0:
             refs.insert(0, Ref.directory(uri=uri + ':all', name='All Tracks'))
+            refs.insert(1, Ref.directory(uri=uri + ':top', name='Top Tracks'))
```

Comment:
Is there a way to avoid displaying this for artists where it's not available, like uploaded artists? If not, this works too. I'm not really sure what the best experience would be here.

Reply:
Should be possible by just repeating the check in line 168 here.

Reply:
Just made the change, works but found an unrelated bug in the process